core/che-core-db/src/main/java/org/eclipse/che/core/db/cascade/CascadeContext.java

```diff
@@ -8,21 +8,24 @@
  * Contributors:
  *   Codenvy, S.A. - initial API and implementation
  *******************************************************************************/
-package org.eclipse.che.core.db.event;
+package org.eclipse.che.core.db.cascade;
+
+import org.eclipse.che.api.core.ApiException;
 
 /**
- * Context that is used only for sharing the state of
- * the process cascading deleting among subscribers.
+ * Context that is used only for sharing the state
+ * of the cascading operation among subscribers.
  *
  * @author Anton Korneta
+ * @author Sergii Leshchenko
  */
-public class RemovalContext {
-    private Exception cause;
+public class CascadeContext {
+    private ApiException cause;
```

Comment:
The same as throwing REST-api oriented exception from Dao. But we do it. :) And we should rethrow original REST-api oriented exception from another Dao